openapi.json

```diff
@@ -658,7 +658,7 @@
             }
           },
           "notifications": {
-            "anyOf": [
+            "OneOf": [
```

Comment:
@yash2189 Could you share the context for this change? I believe changing this to `anyOf` solved the test issue.